src/vdaf.rs

```diff
@@ -115,8 +115,11 @@ pub trait Client: Vdaf {
 
 /// The Aggregator's role in the execution of a VDAF.
 pub trait Aggregator: Vdaf {
-    /// Initial state of the Aggregator during the Prepare process.
-    type PrepareInit: PrepareStep<Self::OutputShare>;
+    /// State of the Aggregator during the Prepare process.
+    type PrepStep;
+
+    /// The type of messages exchanged among the Aggregators during the Prepare process.
+    type PrepMessage;
```

Comment:
nit:
```suggestion
    type PrepareStep;

    /// The type of messages exchanged among the Aggregators during the Prepare process.
    type PrepareMessage;
```

Reply:
I figured you might hate this :) Fixed, here and elsewhere.